sonar-plugin-api/src/main/java/org/sonar/api/batch/DecoratorContext.java

```diff
@@ -30,7 +30,9 @@
 
 /**
  * @since 1.10
+ * @deprecated since 5.6 no more used since Decorator is deprecated
```

Comment:
sentence is a bit strange. What about @deprecated since 5.6 as Decorator is deprecated ?